tests/home.test.js

```diff
@@ -12,3 +12,83 @@ test('Home renders correctly', () => {
   const wrapper = mount(Home, { store, localVue })
   expect(wrapper.html()).toMatchSnapshot()
 })
+
+describe('pagination', () => {
+  let store, actions, state
+
+  // Create new Vuex store with modified state
+  const modifyState = (newState) => {
+    new Vuex.Store({
+      modules: {
+        people: {
+          ...Object.assign(state, newState),
+          actions,
+          namespaced: true
+        }
+      }
+    })
+  }
+
+  // Create mock Vuex store with default state and mock actions
+  beforeEach(() => {
+    state = {
+      page: 1,
+      next: null,
+      previous: null,
+      results: []
+    }
+    actions = {
+      list: jest.fn()
+    }
+    store = new Vuex.Store({
+      modules: {
+        people: {
+          state,
+          actions,
+          namespaced: true
+        }
+      }
+    })
+  })
+
+  it('should render page number', () => {
+    modifyState({page: 3})
+    const wrapper = mount(Home, { store, localVue })
+
+    expect(wrapper.find('#pageNumber').text()).toBe('3')
+  })
+
+  it('should render first page with previous and next disabled', () => {
+    const wrapper = mount(Home, { store, localVue })
+
+    const previousButton = wrapper.find('#previousPage')
+    const nextButton = wrapper.find('#nextPage')
+
+    expect(previousButton.classes()).toContain('disabled')
+    expect(nextButton.classes()).toContain('disabled')
+  })
+
+  it('should render first page with next enabled given non-null next', () => {
+    modifyState({next: "https://swapi.co/api/people/?page=2"})
```

Comment:
These tests are fine but I would have preferred this functionality be tested in Cypress instead of modifying the store directly.